documentation/modules/mirrormaker/proc-mirrormaker-replication.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// assembly-mirrormaker.adoc
+
+[id='proc-mirrormaker-replication-{context}']
+= Synchronizing data between Kafka clusters
+
+Use {MirrorMaker} to synchronize data between Kafka clusters through configuration.
+
+The previous version of {MirrorMaker} continues to be supported.
+If you wish to use the resources configured for the previous version,
+they must be updated to the format supported by {MirrorMaker}.
+
+The configuration must specify:
+
+* Each Kafka cluster
+* Connection information for each cluster, including TLS authentication
+* The replication flow and direction
+** Cluster to cluster
+** Topic to topic
+* Replication rules
+* Committed offset tracking intervals
+
+Use the properties of the `KafkaMirrorMaker2` resource to configure your Kafka {MirrorMaker} deployment.
+
+You can configure access control for source and target clusters using TLS or SASL authentication.
+This procedure shows a configuration that uses TLS encryption and authentication for the source and target cluster.
```

Comment:
@scholzj - I haven't added the configuration for this, but it would be good to have it. Not sure what was needed.